frontend/src/Onboarding/RedirectScreen.tsx

```diff
@@ -32,98 +31,63 @@ export const RedirectScreen: React.FC<Props> = ({ redirectUrl }) => {
-      return academicYear;
-    } else {
-      return 1;
-    }
-  };
-
-  // component did mount
   useEffect(() => {
     setIsLoading(true);
     setIsError(false);
-    fetchMajorsAndPlans(history)(dispatch).then(majors => {
-      const cookie = Cookies.get(AUTH_TOKEN_COOKIE_KEY);
-      if (cookie) {
-        Cookies.remove(AUTH_TOKEN_COOKIE_KEY, {
-          path: "/redirect",
-          domain: window.location.hostname,
-        });
-        // remove cookie if it already exists
-        Cookies.remove(AUTH_TOKEN_COOKIE_KEY, {
-          path: "/",
-          domain: window.location.hostname,
-        });
-        setAuthTokenAsCookie(cookie); // set persisting cookie for all paths
-        fetchActiveUser(cookie)
-          .then(response => {
-            setIsAdvisor(response.user.isAdvisor);
-            if (!response.user.isAdvisor) {
-              // student
+    const cookie = Cookies.get(AUTH_TOKEN_COOKIE_KEY);
+    if (cookie) {
+      Cookies.remove(AUTH_TOKEN_COOKIE_KEY, {
+        path: "/redirect",
+        domain: window.location.hostname,
+      });
+      Cookies.remove(AUTH_TOKEN_COOKIE_KEY, {
+        path: "/",
+        domain: window.location.hostname,
+      });
+      setAuthTokenAsCookie(cookie); // set persisting cookie for all paths
+      fetchActiveUser(cookie)
+        .then(response => {
+          setIsAdvisor(response.user.isAdvisor);
+          if (!response.user.isAdvisor) {
+            // student
 
-              dispatch(setStudentAction(response.user));
-              Promise.all([
-                getScheduleCoursesFromSimplifiedCourseDataAPI(
-                  response.user.coursesCompleted
-                ).then(courses => {
-                  batch(() => {
-                    dispatch(setCompletedCoursesAction(courses));
-                    dispatch(
-                      setStudentAcademicYearAction(
-                        calculateAcademicYear(courses)
-                      )
-                    );
-                  });
-                }),
-                getScheduleCoursesFromSimplifiedCourseDataAPI(
-                  response.user.coursesTransfer
-                ).then(courses => {
-                  dispatch(setTransferCoursesAction(courses));
-                }),
-              ]).then(() => {
-                setNeedsToGoToOnboarding(
-                  !response.user.graduationYear || !response.user.academicYear
-                );
-                setIsLoading(false); // this update must come last, to make sure other state variables are correctly set before we redirect
-              });
-            } else {
-              dispatch(setAdvisorAction(response.user));
-              setIsLoading(false);
-            }
-          })
-          .catch(e => {
-            // TODO: Log error to some service like rollbar
-            console.log(e);
-            setIsError(true);
-          });
-      }
-    });
-  }, []);
+            dispatch(setStudentAction(response.user));
```

Comment:
didn't actually change anything much here, it's mainly whitespace. just removed an unused API call. if you view diff with whitespace togged off, it's a lot more readable